src/main/java/com/beerjournal/BeanConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.beerjournal;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.beerjournal.datamodel.repository.CollectableObjectsRepository;
+import com.beerjournal.datamodel.repository.UserCollectionRepository;
+import com.beerjournal.datamodel.repository.mongo.CollectableObjectsRepositoryImpl;
+import com.beerjournal.datamodel.repository.mongo.UserCollectionRepositoryImpl;
+
+@Configuration
+public class BeanConfig {
+	@Bean
+	public CollectableObjectsRepository getCollectibleObjectsRepository() {
```

Comment:
https://en.wikipedia.org/wiki/Collectable - przyznaję że trochę dziwnie to brzmi nawet jak dla mnie, jednak jest to forma poprawna - jeżeli bardzo kłuje po oczach to mogę zmienić ;)